Gemfile

```
@@ -5,13 +5,16 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.7.5'
 gem 'bootsnap', require: false
+gem 'devise_token_auth', '~> 1.2.1'
 gem 'pg', '~> 1.1'
 gem 'puma', '~> 5.0'
 gem 'rails', '~> 7.0.2', '>= 7.0.2.3'
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 
 group :development, :test do
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'faker', '~> 1.6', '>= 1.6.6'
+  gem 'rspec-json_expectations', '~> 2.2'
```

```suggestion
  gem 'rspec-json_expectations', '~> 2.2.0'
```
Otherwise we are allowing versions like `2.3.x` or `2.4.x` that might contain breaking changes in the future